administrative-sdk/choiceRecognition.js

```diff
@@ -273,91 +245,68 @@ class ChoiceRecognition {
    * @param {Sdk~getChoiceRecognitionCallback} [cb] The callback that handles the response.
    * @param {Sdk~getChoiceRecognitionErrorCallback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.